### DISCUSSION TITLE
想要玲妹的声音吗？！RVC模型分享

### DISCUSSION THREAD

**Discussion by inkluv** [share121/inter-knot#1070] [其他]
### **玲妹RVC模型 无偿分享**
[如果你不会使用RVC技术来制作AI变声，点我](https://www.bilibili.com/video/BV1pm4y1z7Gm)
[常见问题（伸手党点我）](https://www.baidu.com/)

这是本人自己训练的 _玲妹_ 的**RVC模型**
如果你不会使用请你 _点最上方的链接_

**模型信息**

- 音源:玲 - 《绝区零》

- EPOCH:忘了

- STEPs:忘了

反正什么都忘了，就有个模型（

**下载链接↓↓↓↓**
[模型下载点我](https://www.123pan.com/s/tYwhjv-D94D3.html)
**下载链接↑↑↑↑**

提取码：zzzz

不要用模型做奇怪的事情！谢谢配合！

**_效果不好不要怪我，随便训练的 ：）_**

**Comment by inkluv** [comment]
**有问题在这里回复哦！谢谢！XD**

**Comment by Qiling-XwX** [comment]
谢谢楼主！

**Comment by inkluv** [comment]
忘了说了，压缩包文件里面有模型和INDEX，不要漏掉！！

**Comment by LKT142323** [comment]
谢谢楼主!
